### PR TITLE
API-1893: Add guided tutorial "How to retrieve PIM structure"

### PR DESCRIPTION
**Pull request by LevFlavien**
https://akeneo.atlassian.net/browse/API-1893

### CODE REVIEW

File: content/apps/how-to-retrieve-pim-structure.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+# How to retrieve PIM structure
```
Comment (tomglvng) [inline content/apps/how-to-retrieve-pim-structure.md:1]: The content is not up-to-date. See the Notion for modification

File: styles/components.html [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+                <li><a href="urllink">How to parse product values tutorial</a></li>
+                <li><a href="urllink">How to saucisse</a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+
+</body>
+<style>
+    /*table {*/
```
Comment (tomglvng) [inline styles/components.html:88]: Not in CSS file? Why?

File: styles/components.html [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
+    .block-requirements ul {
+        margin: 0;
+        list-style-type: none;
+        padding-left: 60px;
+    }
+    .block-requirements-row {
+        display: flex;
+        justify-content: space-between;
+
+        /*margin-top: 32px;*/
```
Comment (tomglvng) [inline styles/components.html:170]: Remove dead code

File: styles/components.html [hunk 4]
```diff
@@ -0,0 +1,388 @@
...
+        font-family: Lato;
+        font-weight: 600;
+        font-style: italic;
+        text-rendering: optimizeLegibility;
+        src:
+                url("../dist/fonts/lato-bold-italic.woff2") format("woff2"),
+                url("../dist/fonts/lato-bold-italic.woff") format("woff");
+    }
+</style>
+</html>
```
Comment (tomglvng) [inline styles/components.html:388]: Add new line

File: styles/components.html [hunk 5]
```diff
@@ -0,0 +1,388 @@
+<html>
+<head>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+<div>
+    <a href="https://google.com" class="back-button">
```
Comment (tomglvng) [inline styles/components.html:7]: We won't use it for the moment

File: styles/components.html [hunk 6]
```diff
@@ -0,0 +1,388 @@
+<html>
```
Comment (tomglvng) [inline styles/components.html:1]: I don't think a *.html file should be in styles order. WDYT?

File: content/apps/how-to-retrieve-pim-structure.md [hunk 7]
```diff
@@ -45,43 +45,40 @@ Retrieve the PIM Structure through a Channel resource. This is usually the requi
...
 
-$pimUrl = 'https://demo.cloud.akeneo.com';
-$appToken = 'app_token'; // Token provided during oauth steps
+$pimUrl = 'https://url-of-your-pim.com';
+$appToken = 'your_app_token'; // Token provided during oAuth steps
 
 $client = new \GuzzleHttp\Client([
     'base_uri' => $pimUrl,
-    'headers' => ['Authorization' => 'Bearer ' . $appToken],
+		'headers' => ['Authorization' => 'Bearer ' . $appToken],
```
Comment (LevFlavien) [inline content/apps/how-to-retrieve-pim-structure.md:71]: spacing to fix

File: styles/components.html (hunk 5 above)
Comment (LevFlavien) [reply styles/components.html:7]: marking as wip

File: styles/components.html (hunk 6 above)
Comment (LevFlavien) [reply styles/components.html:1]: moving to new directory

File: styles/components.html (hunk 2 above)
Comment (LevFlavien) [reply styles/components.html:88]: made a css file

File: content/apps/how-to-retrieve-pim-structure.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
+# How to retrieve PIM structure
+
+Retrieve the PIM Structure through a Channel resource. This is usually the required step before starting reading the PIM catalog data.
```
Comment (tomglvng) [inline content/apps/how-to-retrieve-pim-structure.md:3]:
```suggestion
Retrieve the PIM structure through a channel resource. This is usually the required step before starting to read the PIM catalog data.
```
Grammarly :)

**Review by tomglvng — APPROVED** [review]
GTM after commiting suggestion

**Review by tseho — APPROVED** [review]

**Pull request merged**